rpsl-parser/irr_test.go

```diff
@@ -746,10 +752,29 @@ func TestParse(t *testing.T) {
 		}
 
 		r := NewReader(fd)
-		got, err := Parse(r)
-		if err == io.EOF {
+		rc := make(chan *Record, 10)
```

Comment:
may want to think about putting an example snippet in the readme (or in an example file) just to show a sample client for the reader ...

Reply:
Oh, this is a good point. 
I'll update the readme to include a quick client.

Reply:
done